Added deprecation notice block above changelog

marians opened the pull request:
Ping @kordless @puja108 @ZeissS : I thought it was a good idea to keep track of upcoming API changes we know of.

What I find sub-optimal is the fact that "some time soon" is not very precise. "by the end of June" or "by June 30" would be better, I think. But this also means that we will have to leave things as they are until then. Can we do that?

marians commented:
Screenshot:

![screenshot 2015-06-03 08 02 32](https://cloud.githubusercontent.com/assets/273727/7953573/e3b8a548-09c6-11e5-8221-10265d71e664.png)

puja108 commented:
would also like to have a more precise date/time, not sure how this affects machine room, alternative would be versioning the API, but I guess here we wouldn't need that, as very few people are using it all anyway.

marians commented:
Since we seem to agree that a clear date makes sense, I modified the text. The old outpute is now guaranteed to exist until June 30. This means we can remove the old stuff somewhere starting July 1st, but not earlier. @ZeissS @ewoutp Please comment if there should be an issue with supporting that.

I also added an additional deprecation notice regarding the old company routes with the same due date.

![image](https://cloud.githubusercontent.com/assets/273727/8023517/06f3379e-0d0f-11e5-9149-e17b8f27beb1.png)

ZeissS commented:
oh, we documtented them? damn, we could have just gotten rid of them. :dancer:

marians commented:
It's not published yet. :)

ZeissS commented:
I meant the routes and the company field. For the last two weeks, I can only identify two users still using the old routes (they were still used in 0.15.0)

marians commented:
OK. Let's see this deprecation notice block as a template we can use once we have stuff to deprecate and email those users directly. Who are they?

ewoutp commented:
I've checked in client-go, the `company` field has a description of `-` which according to the swagger docs means it should not be visible.

Unfortunately it still appears in the api.md file (api project).

So let's consider that a bug for swagger.

marians commented:
Closing unused.
